JetStreamCompose/jetstream/src/main/java/com/google/jetstream/data/util/AssetReader.kt

```diff
@@ -19,7 +19,7 @@ package com.google.jetstream.data.util // ktlint-disable filename
 import android.content.Context
 import com.google.gson.Gson
 
-class AssetsReader(val context: Context, val gson: Gson) {
+class AssetsReader(val context: Context, val gson: Gson = Gson()) {
```

Comment:
In the  `MainActivity`  you are passing  `LocalContext.current`  context which is not application context. It is not safe to pass non application context like that as the lifecycle of the  `AssetsReader`  might outlive the  `MainActivity` lifecycle. I would strongly suggest to use Hilt DI and pass application context here:

`class AssetsReader @Inject constructor(@ApplicationContext private val context: Context, private val gson: Gson)`